webpack.mix.js

```diff
@@ -12,12 +12,26 @@ const mix = require('laravel-mix').mix;
  */
 
 const resources = 'resources/assets';
```

Comment:
But I don't think we need theme as a variable as it is only one place you need to update it. The proxy don't always reflect the theme name.